ggml/src/ggml-sycl/dpct/helper.hpp

```diff
@@ -83,13 +92,32 @@ inline std::string get_device_backend_and_type(const sycl::device &device) {
 }
 
 template <typename Ts> struct matrix_info_t {
-    oneapi::mkl::transpose transpose_info[2];
+    oneapi::math::transpose transpose_info[2];
```

Comment:
Should it rollback to "mkl" for Intel GPU?

Reply:
I have added a namespace alias in https://github.com/ggml-org/llama.cpp/pull/12192/files/0b6f9a90978e79684de0fb5d9129e8f7e2bddcfb#diff-184c9a4290e9a1e26836c54b09c1de9bbf609c50564f21b19fac18f7b74ba25fR25. This means that anything available in `oneapi::mkl` will also be available in `oneapi::math`. This makes it much easier to use both oneMath and Intel oneMKL without more branching everywhere.
`oneapi::mkl` is a deprecated namespace in oneMath so we should avoid using this namespace.